libqtile/widget/clock.py

```diff
@@ -60,6 +50,8 @@ class Clock(base.InLoopPollText):
     def __init__(self, **config):
         base.InLoopPollText.__init__(self, **config)
         self.add_defaults(Clock.defaults)
+        if pytz and isinstance(self.timezone, six.string_types):
```

Comment:
Maybe use `self.timezone is not None`? Maybe log a warning if pytz is not installed?

Reply:
The idea is to let the user pass a tzinfo directly if he has Python 3 but doesn't want to use pytz